actions/publish-github-pages/entrypoint.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+#
+# entrypoint.sh
+#
```

Comment:
We should have a `set -e` where somewhere so we handle error codes

Reply:
If I'm not mistaken, we're doing this here https://github.com/cloudposse/docs/pull/524/files#diff-f3681a161b30981fbebec5e7056818f3eef2d307ccd4d4dceef7e0ee11b38135R14

Reply:
That's the shell for a `run` step, but this is a standalone action with it's own entrypoint and Dockerfile.

Reply:
I was mistaken. Added.